scrapy/utils/reactor.py

```diff
@@ -90,6 +90,24 @@ def verify_installed_reactor(reactor_path):
         raise Exception(msg)
 
 
+def verify_installed_loop(loop_path):
```

Comment:
What do you think about adding "asyncio" to the name? This function seems to be asyncio-specific.

Reply:
Looking at the rest of the code, `verify_installed_asyncio_event_loop` would be the most consistent naming. Would it be too verbose?